### ISSUE TITLE
[Bug] Clock Text is Garbled

### ISSUE THREAD

**Issue by cryptoAlgorithm** [appventure-nush/exam-clock-2020#16]
**Version**
`1.3.B-beta-1.8`

**Describe the bug**
Various UI elements and dialogs in the clock are non-functional or display scrambled text (refer to screenshots and recording below)

**To Reproduce**
* Open the clock (Clock face has garbled text)
* Connect to the clock (Connection dialog has garbled text)

**Expected behavior**
The text and dialogs in the clock are rendered properly

**Screenshots**
![Screenshot 2021-04-30 at 7 40 00 PM](https://user-images.githubusercontent.com/64193267/116690361-db80eb80-a9eb-11eb-8fe2-8e6a745f9b2d.png)
Clock face

![Screenshot 2021-04-30 at 7 37 37 PM](https://user-images.githubusercontent.com/64193267/116690132-87760700-a9eb-11eb-894f-f537b8e5d4b2.png)
Connection dialog

https://user-images.githubusercontent.com/64193267/116690741-695cd680-a9ec-11eb-90ba-528215094740.mov
Screen recording of the issues

**Additional context**
This bug might be caused by JavaFX 16 installed on my system, not rendering the text correctly? This application might be built for an older version of JavaFX.

**Comment by cryptoAlgorithm** [comment]
![Screenshot 2021-04-30 at 7 50 34 PM](https://user-images.githubusercontent.com/64193267/116691397-5696d180-a9ed-11eb-96d4-8b7e2558fb9c.png)
Some dialogs display wrongly

![Screenshot 2021-04-30 at 7 51 00 PM](https://user-images.githubusercontent.com/64193267/116691444-644c5700-a9ed-11eb-83ed-16003ab0d9c3.png)
But some are fine

**Comment by octo-kumo** [comment]
definitely java 16

**Comment by cryptoAlgorithm** [comment]
Yes. But one solution is to get the latest JavaFX library and the issue will be fixed. I cannot remember the exact version but I definitely remember fixing the issue in my own app this way.
